src/app/clusters/groups-server/groups-server.cpp

```diff
@@ -60,107 +66,130 @@
 using namespace chip;
 using namespace chip::app::Clusters;
 using namespace chip::app::Clusters::Groups;
+using namespace chip::Credentials;
 
-static bool isGroupPresent(EndpointId endpoint, GroupId groupId);
+static FabricIndex GetFabricIndex(app::CommandHandler * commandObj)
+{
+    VerifyOrReturnError(nullptr != commandObj, 0);
+    VerifyOrReturnError(nullptr != commandObj->GetExchangeContext(), 0);
+    return commandObj->GetExchangeContext()->GetSessionHandle().GetFabricIndex();
```

Comment:
Oy.  TODO to add a direct accessor for GetFabricIndex() on CommandHandler, please?

Reply:
I have opened a [ticket to add this feature](https://github.com/project-chip/connectedhomeip/issues/11313).